packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenters/datacenters.controller.js

```diff
@@ -33,13 +32,30 @@ export default class {
   }
 
   loadDataCenterDetails(id) {
-    return this.DedicatedCloud.getDatacenterInfoNsxt(
-      this.dedicatedCloud.serviceName,
-      id,
-    ).then(({ data }) => ({
-      edgesCount: data.length,
-      clusterSize: data[0]?.size ? EDGES_SIZES[data[0].size] : '',
-    }));
+    return this.$q
+      .all({
+        vm: this.DedicatedCloud.getDatacenterInfoVm(
+          this.dedicatedCloud.serviceName,
+          id,
+        ),
+        licensed: this.DedicatedCloud.getDatacenterInfoVmLicensed(
+          this.dedicatedCloud.serviceName,
+          id,
+        ),
+      })
+      .then(({ vm, licensed }) => {
+        return {
+          nbreVm: vm?.data?.length || 0,
+          nbreVmLicensed: licensed?.data.length || 0,
+        };
+      })
+      .catch((error) => {
+        return {
+          nbreVm: null,
+          nbreVmLicensed: null,
+          error,
```

Comment:
You don't use the count props so you can remove it
```suggestion
        return {
                  error,
```